tests/addons/TA_fiction_indextime/default/pytest-splunk-addon-data.conf

```diff
@@ -811,4 +811,236 @@ token.0.field = _time
 token.1.token = ##host_token##
 token.1.replacementType = random
 token.1.replacement = host['host']
-token.1.field = host
\ No newline at end of file
+token.1.field = host
+
+# Covers almost all the rules supported by pytest-splunk-addon plugin with a custom index
```

Comment:
Does the test cases fail if you provide a wrong index?

Reply:
Yes, it should fail if it has the wrong index. I have fixed this test issue now.